lib/crowbar/init/application.rb

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+require "/opt/dell/crowbar_framework/lib/crowbar/upgrade_status"
```

Comment:
> I agree with @aspiers we should not hardcode the complete path.

why is a complete path wrong? and don't we always need the complete path anyways, even if we add it to $LOAD_PATH before?

```ruby
LOAD_PATH.push File.expand_path("../../../../../../opt/dell/crowbar_framework/lib", __FILE__)
LOAD_PATH.push File.expand_path("/opt/dell/crowbar_framework/lib", __FILE__)
```